pom.xml

```
@@ -222,7 +222,7 @@
         <asm.version>9.1</asm.version>
         <monitoring-console-api.version>1.2</monitoring-console-api.version>
         <monitoring-console-process.version>1.7</monitoring-console-process.version>
```

Despite the property, we're not in the habit of individually versioning the process vs. the webapp.
```suggestion
        <monitoring-console-process.version>1.8</monitoring-console-process.version>
```